index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fernández-Caminero's Portfolio</title>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+  <header class="artDeco1">
+    <img class="toolbarSize" src="resources/ic_menu.svg" alt="Toolbar Menu">
```

Comment:
- [ Required ] Please consider wrapping this `img` element using the semantic `nav` tag. This will make your webpage more accessible to people using web accessibility tools such as screen readers to identify different elements on a webpage.